cmake/modules/BuildDpdk.cmake

```diff
@@ -0,0 +1,25 @@
+if (CMAKE_SYSTEM_NAME MATCHES "Linux")
+  set(dpdk_env  "x86_64-native-linuxapp-gcc")
```

Comment:
DPDK also compiles on i386 (32 bit) and you enables it by matching `i386|i686`. so `x86_64` does not apply in this case.

Reply:
yes ... this place has problem. Thanks!

Reply:
Fixed in new change, please help take a look.